gooey/gui/widgets/widget_pack.py

```diff
@@ -37,7 +36,12 @@ def onResize(self, evt):
   def get_command(data):
     return data['commands'][0] if data['commands'] else ''
 
-
+  @staticmethod
+  def disable_quoting(data):

```

Comment:
New quoting policy is not applicable if `nargs` is set to `*`, `+` or any other values, except `1` and `?`.